### ISSUE TITLE
Terminal freezes when the number of wav surpasses 15.

### ISSUE THREAD

**Issue by Dimitris1198** [HemantKArya/lsHotword#2]
When  i increase the number of my wavs further than 15 the terminal freezes and nothing happens. I am on windows 10,keras=2.11.0,tensorflow=2.11.0.
I have to remain on those versions cause i can only use cpu.
![image](https://github.com/HemantKArya/lsHotword/assets/99359033/47e9e181-246e-4d98-bce5-da6ca3ece4de)
i dublicated the example wavs and it worked fine.Only with my custome wavs it has an isue. i made mine files sample rate 44.1kHz.
Any ideas on what i am missing?

**Comment by HemantKArya** [comment]
Hi, @Dimitris1198  thanks for reporting this, I will try reproducing this error first. And tell you if something happens.

**Comment by gagipro** [comment]
same. probably wrong input format or something

**Comment by HemantKArya** [comment]
Yeah, that may be reason, but it could have shown an error, I don't why it is freezing in some devices. still can't reproduced it in my setups. any help will be appreciated if anyone knows.

**Comment by EinsPhoenix** [comment]
I encountered the same issue and did some testing. It seems that there's occasionally a problem when inserting the audio and assembling the train.wav file. If this step fails, the entire terminal can freeze, its chance based.

**Comment by EinsPhoenix** [comment]
<img width="1161" height="749" alt="Image" src="https://github.com/user-attachments/assets/a0ea1850-5f86-41b4-a284-71eee2c9c821" />


That was my fix, but i am not an expert on this field and think i broke somethinks with this

**Comment by EinsPhoenix** [comment]
Also is there a reason why you cant do more than 10000 epochs?
